### PR TITLE
ci: Notify users of the aarch64 option when installing

### LINKED ISSUE #360: Update install script help to include `aarch64` option

### PR DESCRIPTION
**Pull request by dark0dave**
Closes #360

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -55,8 +55,8 @@ cat << EOF
 OPTIONS:
    -h      Show help.
    -d      Directory where kubent will be installed. Default is /usr/local/bin
-   -a      Architecture to install (x86_64 only atm.). Default is to auto-detect.
-   -o      OS (linux, macos). Default is to auto-detect.
+   -a      Architecture to install (amd64 or arm64). Default is to auto-detect.
+   -o      OS (linux, macos, windows (windows amd64 only). Default is to auto-detect.
```
Comment (stepanstipl) [inline scripts/install.sh:59]:
```suggestion
   -o      OS (linux, macos, windows [amd64 only]). Default is to auto-detect.
```

Missing closing parenthesis, also use `[]` for the inner ones (although I would maybe leave out the info about arch for windows altogether for clarity).

File: scripts/install.sh [hunk 2]
```diff
@@ -55,8 +55,8 @@ cat << EOF
 OPTIONS:
    -h      Show help.
    -d      Directory where kubent will be installed. Default is /usr/local/bin
-   -a      Architecture to install (x86_64 only atm.). Default is to auto-detect.
-   -o      OS (linux, macos). Default is to auto-detect.
+   -a      Architecture to install (amd64 or arm64). Default is to auto-detect.
```
Comment (stepanstipl) [inline scripts/install.sh:58]:
```suggestion
   -a      Architecture to install (amd64, arm64). Default is to auto-detect.
```
For consistency with the other flags.

**Review by stepanstipl — CHANGES_REQUESTED** [review]
Minor formatting for consistency.

**Comment by dark0dave** [comment]
@stepanstipl updated

**Review by stepanstipl — APPROVED** [review]
👍 thanks

**Pull request merged**